### PR TITLE
REF: implement nanops.na_accum_func

### PR DESCRIPTION
**Pull request by jbrockmendel**
this is a follow-up that was requested a few months ago

### CODE REVIEW

**Comment by jbrockmendel** [comment]
@TomAugspurger thoughts on whether we're going to do a 1.0.3?  if so i should hold off on refactors like this

File: pandas/core/nanops.py [hunk 1]
```diff
@@ -1497,3 +1497,62 @@ def nanpercentile(
             return result
     else:
         return np.percentile(values, q, axis=axis, interpolation=interpolation)
+
+
+def na_accum_func(values, accum_func, skipna: bool):
```
Comment (jreback) [inline pandas/core/nanops.py]: can you add a doc-string

**Review by jreback — COMMENTED** [review]
> @TomAugspurger thoughts on whether we're going to do a 1.0.3? if so i should hold off on refactors like this

this is for 1.1 right?

**Comment by TomAugspurger** [comment]
I'm not sure if we're doing a 1.0.3. Will talk about it on the call today.

File: pandas/core/nanops.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/core/nanops.py]: updated+green

**Comment by jreback** [comment]
thanks

**Pull request merged**